### ISSUE TITLE
Accessibility: Make the buttons in the initial setup screen real buttons with keyboard support

### ISSUE THREAD

**Issue by MarcoZehe** [deltachat/deltachat-desktop#2140]
- Operating System (Linux/Mac/Windows/iOS/Android): All desktop
- Delta Chat Version: 1.14
- Expected behavior: All buttons (Login, sScan QR Code, Restore From Backup) should be buttons and navigable with the tab key, and activable with space or enter.
- Actual behavior: Nothing of the above. They are mere clickable text.
- Steps to reproduce the problem: Make a fresh install of Delta Chat and run it. Try to navigate with the keyboard. Turn on a screen reader and listen to what the "buttons" say, or don't say, namely that they are buttons.
- Screenshots: N/A
- Logs: N/A

## Proposed solution

Either use proper `<button>` elements, which give you all of that for free (preferred), or implement the [button](https://www.w3.org/TR/wai-aria-practices-1.1/#button) design pattern.

**Comment by WofWca** [comment]
This has been mostly fixed by the recent instant onboarding overhaul, but you can't really Load QR code as image using only keyboard.
Also I did not check this with a screen reader, but the buttons are focusable now.

I think this particular issue can be closed, but there is still a lot to do in terms of accessibility.

**Comment by WofWca** [comment]
> but you can't really Load QR code as image using only keyboard.

This is possible since recently.

Please reopen if I missed something.
